<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OOAracAramaFormu</class>
 <widget class="QDialog" name="OOAracAramaFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Renge Göre Arama</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lblArananRenk">
             <property name="text">
              <string>Aranan Renk</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leArananRenk"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="gbRengeGoreAramaSecenekleri">
         <property name="title">
          <string>Arama Seçenekleri</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="rbIleBaslayan">
            <property name="text">
             <string>İle Başlayan</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbIleBiten">
            <property name="text">
             <string>İle Biten</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbIceren">
            <property name="text">
             <string>İçeren</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbYazarkenAra">
            <property name="text">
             <string>YazarkenAra</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btnAra">
           <property name="text">
            <string>Ara</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Giriş Tarihine Göre Arama</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Araç No</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Plaka</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Renk</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Giris Tarihi Saati</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Düzelt</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sil</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Kapat</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>OOAracAramaFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
